model/system_metrics.go

```diff
@@ -32,6 +32,27 @@ var (
 	systemMetricsArtifactKey    = bsonutil.MustHaveTag(SystemMetrics{}, "Artifact")
 )
 
+// CreateSystemMetrics is the entry point for creating the metadata for
+// system metric time series data for a task execution.
+func CreateSystemMetrics(info SystemMetricsInfo, artifact SystemMetricsArtifactInfo) *SystemMetrics {
+	artifact.Prefix = info.ID()
+	artifact.Key = "system_metrics"
```

Comment:
Maybe we should have a way of knowing if the file has been uploaded to s3 or not? Or have a slice of keys instead?